### ISSUE TITLE
Same JSON input produces broken SVG with NPM module vs website

### ISSUE THREAD

**Issue by Hades32** [sverweij/state-machine-cat#39]
### Used Version: 4.4.6

This model gives different outputs when I'm using it via the NPM module vs when I try it on the official website - see screenshots

### Model:
[model.txt](https://github.com/sverweij/state-machine-cat/files/2872401/model.txt)

### used code
```
        let lSVGInAString = smcat.render(data, {
          inputType: "smcat-json", //unknown value actually leads to pass through. data is pre-parsed JSON
          outputType: "svg",
          direction: "left-right"
        });
        document.getElementById("svg-stama").innerHTML = lSVGInAString;
```

### Good Rendering:
![good render](https://user-images.githubusercontent.com/167847/52905517-94a7e080-323b-11e9-9d76-8080bccf4369.JPG)

### Bad Rendering:
![bad render](https://user-images.githubusercontent.com/167847/52905518-94a7e080-323b-11e9-95b6-4c5df6c97e1c.JPG)

I'm unsure if I'm doing something wrong or the code is truly different...

**Comment by Hades32** [comment]
BTW I want to use your great package to eventually visualize my current state machine states:
https://github.com/Hades32/stama-visualizer

**Comment by sverweij** [comment]
@Hades32 thanks for raising this issue. 

I've found the culprit.
- The short term solution is to explicitly pass `"dot"` as `engine`. 
`dot` is  the default engine, so you shouldn't have to pass it, but there's one piece in the code not yet up to speed with that :-/. The reason it works on the website is that that always passes the engine.
- As an alternative you can use the beta I've just published on npm (`state-machine-cat@4.4.7-beta-2`) that fixes the issue. 

Tonight I hope to promote the beta to a regular (4.4.7) release

B.t.w. you can use `"json"` as `inputType`. Currently `"smcat-json"` doesn't break anything, but it's not guaranteed to work as it's not a known input type (either `smcat` or `json`).

Your stama-visualizer is a cool application of state-machine-cat :-)

**Comment by Hades32** [comment]
wow, that was fast! :)

Thank you, I updated both properties as you suggested and now everything is looking great!
